include/RAJA/policy/cuda/kernel/internal.hpp

```diff
@@ -259,16 +286,23 @@ struct KernelDimensionCalculator<RAJA::policy::cuda::cuda_indexer<
   using IndexMapper = cuda::IndexGlobal<dim, BLOCK_SIZE, named_usage::ignored>;
 
   template<typename IdxT>
-  static void set_dimensions(CudaDims& dims, CudaDims& min_dims, IdxT len)
+  static LaunchDims get_dimensions(IdxT len)
   {
+    constexpr auto zero = static_cast<IdxT>(0);
+
     if (len > static_cast<IdxT>(IndexMapper::block_size))
     {
       RAJA_ABORT_OR_THROW(
           "len exceeds the size of the directly mapped index space");
     }
-    set_cuda_dim<dim>(dims.threads, static_cast<IdxT>(IndexMapper::block_size));
-    set_cuda_dim<dim>(min_dims.threads,
-                      static_cast<IdxT>(IndexMapper::block_size));
+
+    LaunchDims dims;
+
+    set_cuda_dim<dim>(dims.  active.threads, static_cast<cuda_dim_member_t>(true));
+    set_cuda_dim<dim>(dims.    dims.threads, static_cast<cuda_dim_member_t>((len > zero) ? IndexMapper::block_size : 0));
```

Comment:
Can you use "zero" instead of "0"?

Reply:
Oh, I didn't realize it was an int in these cases. I don't think it really adds any clarity to make changes, so it's probably not worth worrying about.